src/components/Sep6/Sep6Deposit.tsx

```diff
@@ -108,53 +133,86 @@ export const Sep6Send = () => {
   if (sep6DepositAsset.status === ActionStatus.NEEDS_INPUT) {
     return (
       <Modal visible={true} onClose={handleClose}>
-        <Heading2
-          className="ModalHeading"
-          tooltipText={
-            <>
-              These are the fields the receiving anchor requires. The sending
-              client obtains them from the /customer endpoint.{" "}
-              <TextLink
-                href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#customer-get"
-                isExternal
-              >
-                Learn more
-              </TextLink>
-            </>
-          }
-        >
-          SEP-6 Required Info
-        </Heading2>
         <div className="ModalBody">
-          {Object.entries(sep6DepositAsset.data.depositTypes || {}).map(
-            ([id, input]) => (
+          <Heading2
+            className="ModalHeading"
+            tooltipText={
               <>
-                <Select
-                  label={input.description}
-                  id={id}
-                  key={id}
-                  onChange={handleDepositTypeChange}
+                These are the fields the receiving anchor requires. The sending
+                client obtains them from the /info endpoint.{" "}
+                <TextLink
+                  href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#info"
+                  isExternal
```

Comment:
I'm also differentiating between fields required by SEP-12 and fields required by SEP-6